css/style.css

```diff
@@ -500,6 +981,157 @@
     background-color: #e51937;
     color: #fff;
   }
+
+  .wrap-information {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: column;
+    background-position-y: top;
+    height: 1125px;
+    margin-top: 0;
+  }
+
+  .info__title {
+    text-align: right;
+    padding: 0 130px 0 0;
+    margin-top: 45px;
+  }
+
+  .red-first {
+    max-width: 1023px;
+    height: 530px;
+    float: none;
+    padding: 63px 47px 0 0;
+  }
+
+  .redrow {
+    flex-direction: row;
+    display: flex;
+    height: 300px;
+    padding-bottom: 10px;
+  }
+
+  .description, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)